deisctl/backend/backend.go

```diff
@@ -1,14 +1,25 @@
 package backend
 
-import "sync"
+import (
+	"io"
+	"sync"
+)
 
 // Backend interface is used to interact with the cluster control plane
 type Backend interface {
-	Create([]string, *sync.WaitGroup, chan string, chan error)
-	Destroy([]string, *sync.WaitGroup, chan string, chan error)
-	Start([]string, *sync.WaitGroup, chan string, chan error)
-	Stop([]string, *sync.WaitGroup, chan string, chan error)
-	Scale(string, int, *sync.WaitGroup, chan string, chan error)
+	/*
+		Create([]string, *sync.WaitGroup, chan string, chan error)
+		Destroy([]string, *sync.WaitGroup, chan string, chan error)
+		Start([]string, *sync.WaitGroup, chan string, chan error)
+		Stop([]string, *sync.WaitGroup, chan string, chan error)
+		Scale(string, int, *sync.WaitGroup, chan string, chan error)
+	*/
```

Comment:
Could you remove this comment? Version control should be sufficient to document the previous interface.